src/routes/jira/events/jira-events-install-post.ts

```diff
@@ -10,12 +10,19 @@ export const JiraEventsInstallPost = async (req: Request, res: Response): Promis
 	req.log.info("Received installation payload");
 
 	const { baseUrl: host, clientKey, sharedSecret } = req.body;
-	await Installation.install({
+	const newOrExistingInstallation = await Installation.install({
 		host,
 		clientKey,
 		sharedSecret
 	});
 
+	if (!newOrExistingInstallation.plainClientKey) {
```

Comment:
I'm not sure I understand, why isn't this part of the model itself?  This is model logic, it doesn't matter where the `install` call comes from, it should always add the `plainClientKey` to the DB...

Reply:
just add it to the install.  Better to have the logic centralized to the model instead of bleeding out in the webhook logic.  Like for instance, how I'm currently changing the webhooks...